mlflow/models/evaluation/base.py

```diff
@@ -1316,6 +1317,25 @@ def evaluate(
         .. _textstat:
             https://pypi.org/project/textstat
 
+    - For retriever models, it's recommended to use a static dataset represented by a Pandas
+      Dataframe or an MLflow Pandas Dataset containing the input queries, retrieved relevant
```

Comment:
nit, can we make sure this docs page renders properly 
<img width="1227" alt="image" src="https://github.com/mlflow/mlflow/assets/46332835/6785e5a4-8215-40f7-9966-64c789e58b19">

Reply:
Fixed 
<img width="653" alt="Screenshot 2023-10-23 at 5 51 17 PM" src="https://github.com/mlflow/mlflow/assets/7851093/f2e8075d-b589-4e4f-9d44-c45f8e8d7fdf">